### ISSUE TITLE
Content: Update FAQ for legal advice

### ISSUE THREAD

**Issue by SamHyler** [hackforla/expunge-assist#1597]
### Overview
UXR has indicated that some parts of the FAQ can be construed as legal advice. It is important we do not have legal advice on our website. This is the first step in improving the FAQs.

### Action Items
- [x] https://github.com/hackforla/expunge-assist/issues/1598 UXR completes exercise on FigJam https://www.figma.com/board/DQcMAN49OUsCieINIZbAh0/UXR-FigJam-board---Expunge-Assist?node-id=1177-5860&t=l6sQxDWz8GCG5TVr-4
- [x] Discuss in Content
- [x] Remove areas that can be construed as legal advice in Figma
- [x] Discuss with Design on any spacing issues, create issue if necessary
- [x] Update Figma Master pages (Desktop, Mobile) 
- [x] Update Content SOT (source of truth) in the sheets doc
- [x] handoff to Dev #1641
   - [ ] add deliverable(design or content)
   - [ ] remove https://github.com/hackforla/expunge-assist/labels/dependency label
   - [ ] add https://github.com/hackforla/expunge-assist/labels/ready%20for%20dev%20lead label
   - [ ] move to `new issue approval` column 

### Resources/Notes
UXR exercise on FigJam https://www.figma.com/board/DQcMAN49OUsCieINIZbAh0/UXR-FigJam-board---Expunge-Assist?node-id=1177-5860&t=l6sQxDWz8GCG5TVr-4

Figma page for WIP https://www.figma.com/design/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=2-23

**Comment by SamHyler** [comment]
2 FAQs are considered legal advice https://github.com/hackforla/expunge-assist/issues/1598

**Comment by SamHyler** [comment]
Prototype complete, tagged Design for review @anitadesigns @jyehllow 

https://www.figma.com/design/hYqRxmBVtJbDv9DJXV6nra/Expunge-Assist-Main-Figma?node-id=2-23

**Comment by SamHyler** [comment]
Disucssed in Content. Waiting on final Design check go-ahead, then will start handover process and update all areas for SOT

**Comment by SamHyler** [comment]
Ready, need to update master and copydoc SOT hotspots. Have copied in to Dev issue, need to discuss new/old handover proceedures

**Comment by SamHyler** [comment]
Updates master pages, SOT copydoc, issue for dev created. Has been assigned so assume everything was smooth there.
